CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.6] - 2020-07-25
```

Comment:
```suggestion

```
We keep all the changes under `Unreleased` section and only when we want in fact to create a new release, we move them to the proper release title, like this https://github.com/scanapi/scanapi/pull/212/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR10

So for now, we can leave everything under ##[Unreleased]